aws/csm/reporter_test.go

```diff
@@ -147,7 +147,7 @@ func TestReportingMetrics(t *testing.T) {
 				req := request.New(*sess.Config, md, sess.Handlers, client.DefaultRetryer{NumMaxRetries: 3}, op, nil, nil)
 				errs := []error{
 					awserr.New("AWSError", "aws error", nil),
-					awserr.New("RequestError", "sdk error", nil),
+					awserr.New(request.ErrCodeRequestError, "sdk error", nil),
```

Comment:
Why is this change specifically for "RequestError" ? 

Should we do the same for other errors as well like the AWSError here.